docs/DeveloperGuide.adoc

```diff
@@ -729,9 +729,87 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |set certain contacts as favourites |access their information easily
+
+|`* * *` |user |select multiple contacts to delete |clean up address book efficiently
+
+|`* * *` |user |save more than one contact number per contact in my address book |account for multiple phone numbers
+
+|`* * *` |organised user |group contacts manually or automatically |access the information I want easily
+
+|`* * *` |user with busy schedules |see my important events on a calendar view |have an overview of my schedule for the month
+
+|`* * *` |organized user |share my contact details with my friends |save the contacts without manually keying in the details
+
+|`* * *` |impatient user |sort the information on the application |view the information in a ordered manner
+
+|`* * *` |user |add information to the application in a intuitive way |feel less frustrating
+
+|`* * *` |user |sync my account with other platforms |save the trouble of entering my personal details
```

Comment:
You should be more specific for `other platforms` mentioned.

Reply:
I'm a bit confused about this? How detailed should we go?

Reply:
You can mention the platforms that you want to sync to.